CandidateExercise/ConfigurationExcerciseTests/ConfigValueTests.cs

```diff
@@ -0,0 +1,91 @@
+﻿using Configuration_Exercise;
+using NUnit.Framework;
+
+namespace FutureWonder.Exercises.Configuration.ConfigValueTests
+{
+    [TestFixture, Category("ConfigValue"), Parallelizable(ParallelScope.All)]
+    public class ConfigValueTests
+    {
+        private const int BasicInt = 256;
+        private const double BasicDouble = 2147483648;
```

Comment:
I would suggest using a double with decimal places, or adding a second double to test alongside your BasicDouble (such as "DecimalDouble = 34.437").

In my experience, those decimal places can cause big problems in code using doubles, so it would be great to have test coverage.

Reply:
Good point